src/main/java/org/mozilla/taskcluster/client/temporarycredentials/Certificate.java

```diff
@@ -0,0 +1,130 @@
+package org.mozilla.taskcluster.client;
```

Comment:
The package doesn't match the directory - let's put these files in the client directory (i.e. up a directory) and leave the package name as you have it.